pom.xml

```diff
@@ -28,61 +28,37 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.21</version>
+    <version>4.15</version>
   </parent>
 
   <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
-  <artifactId>icon-shim-pom</artifactId>
+  <artifactId>icon-shim</artifactId>
   <version>2.0.4-SNAPSHOT</version>
-  <packaging>pom</packaging>
+  <packaging>hpi</packaging>
 
-  <name>Icon Shim Base POM.</name>
-  <description>
-    Base POM for the Icon Shim modules. The hope is that these modules (two of) should collapse into one (i.e. just a
-    plugin module) if Jenkins core can lose it's dependency on the icon-set module. This should be possible if we
-    change the core icon tag to use CSS based icons.
-  </description>
+  <name>Icon Shim</name>
+  <description>Deprecated: No longer does anything</description>
   <url>https://github.com/jenkinsci/icon-shim-plugin</url>
 
   <licenses>
     <license>
       <name>MIT License</name>
-      <url>http://opensource.org/licenses/MIT</url>
+      <url>https://opensource.org/licenses/MIT</url>
     </license>
   </licenses>
 
-  <modules>
-    <module>icon-set</module>
-    <module>plugin</module>
```

Comment:
To keep the diff here smaller, I would have suggested leaving the parent POM alone and just updating `plugin`/, but OK.